flink-connectors/flink-connector-cassandra/pom.xml

```diff
@@ -83,6 +85,10 @@ under the License.
 										<exclude>com.google.inject.**</exclude>
 									</excludes>
 								</relocation>
+								<relocation>
+									<pattern>io.netty</pattern>
+									<shadedPattern>org.apache.flink.shaded.netty4.io.netty</shadedPattern>
```

Comment:
Thanks for your review and reply! I see your points now, and I still have a few questions to the shading approach though:
1. There seems to be some policies for making shadedPattern for different scenarios. Would there be any external documents that possibly address these rules?
2. It is obvious to shade a *direct dependency, since we need it to be free of class clashes. How would we decide which *indirect dependencies to be shaded?
3. The benefit of shading is clear, but wouldn't it increase the jar file size if there are more dependencies that need to be managed later on - also the effort of managing the including/relocation rules.

I will update another version of PR later. Thanks!